src/Forms/SearchableDropdownTrait.php

```diff
@@ -225,6 +224,9 @@ public function setUseDynamicPlaceholder(bool $useDynamicPlaceholder): static
 
     /**
      * Get whether to use a search context instead searching on labelField
+     *
+     * If labelField is is set to a getter method on the DataObject e.g. 'Title' => getTitle(),
+     * then search context will be used regardless of what is returned here
```

Comment:
```suggestion
     * If the label field doesn't map to a database field (e.g. 'MyField' which is backed by a 'getMyField()' method)
     * then search context will be used regardless of what is returned here
```
What matters is whether it maps to a database field or not. There can be a getter method _and_ a database field, in which case the label would still be used.

Please update the other PHPDocs as well.

Reply:
Have changed the wording a bit because was a confusing to me, have updated other docs too